<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheatMenu</class>
 <widget class="QDialog" name="CheatMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CheatMenu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="cheatFrame">
     <property name="styleSheet">
      <string notr="true">border-image: url(:/cheat/cheatBackground.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
      <item>
       <widget class="QLabel" name="cheatTitle_2">
        <property name="font">
         <font>
          <pointsize>21</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: none; color: red;</string>
        </property>
        <property name="text">
         <string>Worms2D Cheat Menu</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="ww3Button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>WW3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="noWindButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>No Wind</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="infiniteAmmoButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Infinite Ammo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="supplyRunButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Supply Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="inmortalWormButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Immortal Worms</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="multipleJumpButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Multiple Jump</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="infiniteTurnButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Infinite Turn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="everyone1HPButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/cheat/buttonBorder.png);
color: rgb(255, 58, 255)</string>
            </property>
            <property name="text">
             <string>Everyone 1HP</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
